docs/keyd-application-mapper.scdoc

```diff
@@ -62,9 +74,10 @@ E.G:
 Will remap _A-1_ to the the string 'Inside st' when a window with a class
 that begins with 'st-' (e.g st-256color) is active. 
 
-Window class and title names can be obtained by inspecting the log output while the
-daemon is running (e.g _tail\ -f\ ~/.config/keyd/app.log_). A reload may be triggered
-by sending the script a USR1 signal.
+Window class and title names can be obtained by inspecting the output if the program
+was started with the _-v, --verbose_ option. If the program was daemonized, you can
+read the log file output by e.g. _tail\ -f\ ~/.config/keyd/app.log_.
+A reload may be triggered by sending the script a USR1 signal.
```

Comment:
another thing: it appears that on my machine (`sway`), the `USR1` signal just stops the process. Looking at the script, it looks like the function that attempts to catch USR1 is only used in the GNOME Monitor and XMonitor classes, and not in the others including Wayland and Wlroots. Is there some difference among window managers that affects how the process should handle signals? I would imagine not. Might raise a separate issue about this

Reply:
#935